app/middleware/translator.py

```diff
@@ -27,10 +25,9 @@ class JSONTranslator(object):
     def process_request(self, req, res):
         if self.__get_content_type(req) == 'application/json':
             try:
-                raw_json = req.stream.read()
+                raw_json = req.bounded_stream.read()
```

Comment:
req.stream class is as follows.

v1.4.1
- gunicorn run
gunicorn.http.body.Body
- test simulator
falcon.request_helpers.BoundedStream

v3.0.1
- gunicorn run
gunicorn.http.body.Body
- test simulator
wsgiref.validate.InputWrapper

InputWrapper.read() requires parameters and causes an error.
Used BoundedStream.
ref https://falcon.readthedocs.io/en/stable/api/request_and_response_wsgi.html#falcon.stream.BoundedStream
curl command, requests Library, FetchAPI, and axios Library sets Content-Length header automatically, so it seems to have no influence.